### ISSUE TITLE
Switch Pro Controller random Inputs

### ISSUE THREAD

**Issue by TheAgamer554** [isXander/Controlify#79]
**Describe the bug**
When I open Minecraft with the mod and a Switch Pro Controller plugged in I get random inputs and I can't do anything

**To Reproduce**
1. Plug in Switch Pro Controller
2. Open Minecraft with the mod enabled (lastest release)
3. I get random inputs and it keeps turning on and off features, changing the menus and stuff

**Expected behavior**
The controller should work properly

**Screenshots**
![imagen](https://github.com/isXander/Controlify/assets/42060301/6f99b3e0-b164-4cbc-ae60-c4add39ab3ff)

**Minecraft Info (please complete the following information):**
 - OS: [e.g. Windows 10]
 - Minecraft Version: [1.19.4]
 - Mod Version: [1.2.1+1.19.4]

**Additional context**

**Comment by isXander** [comment]
Are you sure this is a Controlify problem and not your controller? How have you tested this?

**Comment by TheAgamer554** [comment]
It is a Switch Pro controller, on aplications that use DInput a similar issue occurs, but with programs that use SDL2 like Dolphin, Cemu or Steam the controller works perfectly. I assumed it would work using SDL2 since that's what's generally used to gather data on gyro which I thought was supported

**Comment by Crackers0106** [comment]
I think that this might be a Windows issue and not a Controlify one. I've had this same problem on MidnightControls and other games that aren't Minecraft, and when I open the Windows control panel controller options it does it there too.
I remember I was able to fix it by doing some messing around with my operating system and trying to connect my controller in a different way, but I don't remember what I did exactly to fix it.
